RELEASENOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## v2.5.5
+### Fixes
+ - Problem with Exoplayer default implementation of methods on interfaces (#162)
```

Comment:
This is probably fine, but I think we should be a bit more careful with PR names if we're going to copy PR names directly into the release notes. This doesn't explain what's really broken, so if I'm skimming the release notes, it's not clear that this would fix my AbstractMethodException or whatever it's called. We can keep this for now, but I'd love to see us either enforce a scheme to make sure PRs are more descriptive of what they fix, or manually write out what issues this type of thing fixed.